nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<p>Please remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>